reconcile/aws_cloudwatch_log_retention/integration.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import re
+from collections.abc import Callable
+from typing import Optional
+
+from pydantic import BaseModel
+
+from reconcile import queries
+from reconcile.queries import get_aws_accounts
+from reconcile.utils.aws_api import AWSApi
+
+QONTRACT_INTEGRATION = "aws_cloudwatch_log_retention"
+MANAGED_TAG = {"Key": "managed_by_integration", "Value": QONTRACT_INTEGRATION}
+
+
+class AWSCloudwatchLogRetention(BaseModel):
+    name: str
+    acct_uid: str
+    log_regex: str
+    log_retention_day_length: str
+
+
+def get_app_interface_cloudwatch_retention_period(aws_accounts: list) -> list:
```

Comment:
get_app_interface_cloudwatch_retention_period now should get single `aws_acct` instead of `aws_accounts` list.